### PR TITLE
Add only nodes available from toolbox

### PR DESCRIPTION
**Pull request by jedrz**
Unknown nodes or available in other categories cannot be added to process.

### CODE REVIEW

File: ui/client/common/ProcessDefinitionUtils.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    return {
+      ...group,
+      possibleNodes: getPossibleNodesInCategory(group.possibleNodes, category)
+    }
+  })
+}
+
+const getFlatNodesToAddInCategory = (processDefinitionData, category) => {
+  const nodesToAdd = getNodesToAddInCategory(processDefinitionData, category)
+  return _.flatMap(nodesToAdd, group => getPossibleNodesInCategory(group.possibleNodes, category))
```
Comment (mproch) [inline ui/client/common/ProcessDefinitionUtils.js]: you don't need getPossibleNodesInCategory here? They are already filtered?

File: ui/client/test/ProcessDefinitionUtils-test.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  it("should return all nodes available in category", () => {
+    const nodesToAdd = ProcessDefinitionUtils.getNodesToAddInCategory(processDefinition, "Category1")
+
+    expect(nodesToAdd).toEqual(processDefinition.nodesToAdd)
+  })
+
+  it("should filter out unavailable nodes in category", () => {
+    const nodesToAdd = ProcessDefinitionUtils.getNodesToAddInCategory(processDefinition, "Category2")
+
+    expect(nodesToAdd).toEqual([
```
Comment (mproch) [inline ui/client/test/ProcessDefinitionUtils-test.js]: can we make this assertion a bit more understandable? ;)

File: ui/client/common/ProcessDefinitionUtils.js (hunk 1 above)
Comment (jedrz) [reply ui/client/common/ProcessDefinitionUtils.js]: Yup, you're right.

File: ui/client/test/ProcessDefinitionUtils-test.js (hunk 2 above)
Comment (jedrz) [reply ui/client/test/ProcessDefinitionUtils-test.js]: Done.

**Pull request merged**